bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/internal/dialogs/ViewsPreferencePage.java

```diff
@@ -192,6 +202,28 @@ protected Control createContents(Composite parent) {
 		return comp;
 	}
 
+	private void createHiDPISettingsGroup(Composite parent) {
+		if (!OS.isWindows() || true) {
```

Comment:
The options do not appear anymore because this always evaluates to `true`, such that the method is never executed.

Reply:
Sorry, leftover...